examples/generalized_linear_models/GLM-discrete-choice_models.myst.md

```diff
@@ -0,0 +1,772 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc_examples_new
+  language: python
+  name: pymc_examples_new
+myst:
+  substitutions:
+    extra_dependencies: jax, jaxlib, numpyro
+---
+
+(discrete_choice)=
+# Discrete Choice and Random Utility Models
+
+:::{post} June, 2023
+:tags: categorical regression, generalized linear model, discrete choice 
```

Comment:
should we use `model expansion` here too? it is used in the golf putting example. I think it is great to see this kind of examples where we iterate on a model and improve it (it would even be interesting to see an example where we iterate and we often go backwards even but I digress). I think this tag could be used as an indicator of these kind of notebooks: https://pymcio--544.org.readthedocs.build/projects/examples/en/544/blog/tag/model-expansion.html

Reply:
I've added this tag too.